uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1025.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1025 {
+
+  /**
+   * Deletes some demographics fields where they are not in use
+   *
+   * @return bool
+   */
+  public function upgrade_1025() {
+
+    $this->deleteExtendDemographicFields([
+      'Ethnicity',
+      'Religion',
+      'Sexual_Orientation',
+    ]);
+
+    return TRUE;
+  }
+
+  /**
+   * Deletes Custom Demographic Fields only if they are not used
+   *
+   * @param $fieldsToDelete
+   */
+  private function deleteExtendDemographicFields($fieldsToDelete) {
```

Comment:
For upgraders we usually start method names with the "up" + version, e.g. `up1025_deleteExtendedDemographicFields`